Gemfile

```diff
@@ -28,9 +28,8 @@ gem 'mongo_session_store', '>= 3.1.0' # stores sessions in database for security
 gem 'enumerize' # Mongoid doesn't have enum out of the box, so we get it here
 
 # Our authentication library is devise, with oauth2 for google signin
-gem 'devise', '~> 4.3'
-gem 'omniauth-google-oauth2', '0.2.1' # TODO upgrade
-gem 'omniauth-oauth2', '1.3.1' # TODO remove this pin
```

Comment:
PREGUNTA: I take it that devise folded omniauth-oauth2 into the main package and thus the drop?

Reply:
We previously had to have this pinned because of a nasty incompatibility while some programmers argued about whose problem it was, or something? Either way, `omniauth-google-oauth2` doesn't break anymore if the `omniauth-oauth2` gem is too high, so we're good to just go with whatever the `omniauth-google-oauth2` gem would like.